src/data/detail-page/tabs/backend.ts

```diff
@@ -1,3 +1,3 @@
-import { categoryId } from './tabsInterfaces';
+import { CategoryId } from './tabsInterfaces';
 
-export const backend: categoryId[] = ['learn', 'jobs', 'resources'];
+export const backend: CategoryId[] = ['learn', 'jobs', 'resources'];
```

Comment:
For futher discussion: I see that all category files from tabs directory have identical content. Do we need seperated files with categories for future improvements?

Reply:
Yes, they are the same, but maybe in the future they we would like to make the different or do note have all of them in some specialization. That's why I make it separate for each one.